src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -1,33 +1,53 @@
 /* ==================================================
- * Authors:
+ * Authors: Lemi Miyu and Fawaaz
  *
  * --------------------------------------------------
- * Description:
- *
+ * Description: Drivetrain.java contains functions
+ * and objects needed with the drivetrain's motors
+ * in order to control robot movement
  * ================================================== */
 
 package frc.robot.subsystems;
 
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import edu.wpi.first.wpilibj.motorcontrol.PWMSparkMax;
 
 public class Drivetrain extends SubsystemBase
 {
   // HARDWARE //
 
   // declare motor controllers, solenoids, and other hardware here
 
-  // CONSTANTS //
+  private PWMSparkMax rightMotor;
+  private PWMSparkMax leftMotor;
 
+  // PORTS //
 
+  public static final int RIGHT_MOTOR = -1; // port not final
+  public static final int LEFT_MOTOR = -2; // port not final
+
+  // CONSTANTS //
+
+  
 
   // CONSTRUCTOR //
 
   public Drivetrain()
   {
     // instantiate hardware
+    rightMotor = new PWMSparkMax (RIGHT_MOTOR);
+    leftMotor = new PWMSparkMax(LEFT_MOTOR);
   }
 
   // METHODS //
 
+  public void moveMotors (double speed) {
+    rightMotor.set(speed);
+    leftMotor.set(speed);
+  }
```

Comment:
This only moves both sides of the robot in the same direction right?  We will also need to factor in the turning speed which means that the left and right sides may be spinning in different directions.

Reply:
Check how the drivetrain was implemented in the old code to see how they implemented that with turning.